modules/nw-anp-audit-logging-concept.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/logging-anp-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-anp-audit-logging_{context}"]
+= AdminNetworkPolicy audit logging
+
+Audit logging is enabled per `AdminNetworkPolicy` CR by annotating an ANP policy with the `k8s.ovn.org/acl-logging` key such as in the following example:
+
+.Example of annotation for `AdminNetworkPolicy` CR
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  annotations:
+    k8s.ovn.org/acl-logging: '{ "deny": "alert", "allow": "alert", "pass" : "warning" }'
+  name: anp-tenant-log
+spec:
+  priority: 5
+  subject:
+    namespaces:
+      matchLabels:
+        tenant: backend-storage # Selects all pods owned by storage tenant.
+  ingress:
+    - name: "allow-all-ingress-product-development-and-customer" # Product development and customer tenant ingress to backend storage.
+      action: "Allow"
+      from:
+      - pods:
+          namespaceSelector:
+            matchExpressions:
+            - key: tenant
+              operator: In
+              values:
+              - product-development
+              - customer
+          podSelector: {}
+    - name: "pass-all-ingress-product-security"
+      action: "Pass"
+      from:
+      - namespaces:
+          matchLabels:
+              tenant: product-security
+    - name: "deny-all-ingress" # Ingress to backend from all other pods in the cluster.
+      action: "Deny"
+      from:
+      - namespaces: {}
+  egress:
+    - name: "allow-all-egress-product-development"
+      action: "Allow"
+      to:
+      - pods:
+          namespaceSelector:
+            matchLabels:
+              tenant: product-development
+          podSelector: {}
+    - name: "pass-egress-product-security"
+      action: "Pass"
+      to:
+      - namespaces:
+           matchLabels:
+             tenant: product-security
+    - name: "deny-all-egress" # Egress from backend denied to all other tenants.
+      action: "Deny"
+      to:
+      - namespaces: {}
+----
+====
+
+Logs are generated whenever a specific OVN ACL triggers an event that meets the action criteria set in your logging annotation. For example, an event in which any of the namespaces with the label `tenant: product-development` accesses the namespaces with the label `tenant: backend-storage`, a log is generated.
```

Comment:
https://github.com/openshift/openshift-docs/pull/75310#discussion_r1620326542 this comment has not been fixed.
I still don't follow "OVN ACL triggers" terminology. PTAL.